readthedocs_ext/_static/readthedocs-data.js_t

```diff
@@ -2,14 +2,18 @@
     project: "{{ slug }}",
     version: "{{ version_slug }}",
     language: "{{ rtd_language }}",
+
     subprojects: {},
     canonical_url: "{{ canonical_url }}",
     theme: "{{ html_theme }}",
     builder: "sphinx",
     docroot: "{{ conf_py_path }}",
     source_suffix: "{{ source_suffix }}",
     api_host: "{{ api_host }}",
-    commit: "{{ commit }}"
+    commit: "{{ commit }}",
+
+    analytics_code: {% if global_analytics_code %}'{{ global_analytics_code }}'{% else %}null{% endif %},
```

Comment:
I'm not sure we need this one, since that's likely one we do want defined in the server. Guess it doesn't hurt to include it and not use it though.

Reply:
Actually, you're right. It's probably best to remove it.

Reply:
I ended up deciding this was a good idea in case anyone runs RTD in their own infrastructure. For all builds on readthedocs.org it should be the same.